tests/e2e/yamcs/barGraph.e2e.spec.mjs

```diff
@@ -0,0 +1,94 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2024, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+/*
+ * This test suite is dedicated to testing the Bar Graph component.
+ */
+
+import { pluginFixtures, appActions } from 'openmct-e2e';
+const { test, expect } = pluginFixtures;
+const { createDomainObjectWithDefaults, expandEntireTree } = appActions;
+
+test.describe('Bar Graph', () => {
+  let barGraph;
+  let historicalGet;
+
+  test.beforeEach(async ({ page }) => {
+    // Open a browser, navigate to the main page, and wait until all networkevents to resolve
+    await page.goto('./', { waitUntil: 'domcontentloaded' });
+
+    // Create the Bar Graph
+    barGraph = await createDomainObjectWithDefaults(page, { type: 'Graph' });
+    // Enter edit mode for the overlay plot
+    await page.getByLabel('Edit Object').click();
+  });
+
+  test('Requests a single historical datum', async ({ page }) => {
```

Comment:
Where's the assertion in this test?

Reply:
@akhenry https://github.com/akhenry/openmct-yamcs/pull/470/files/f62b4c54baa53ffda462a91b7814032216ae2b85#diff-c9a10b8751a2a70edf45ed5991bb3728276e81ec7ca795d39724f88265f055b0R51

The assertion is that the `historicalGet` with `limit=1` resolves